scripts/drone/steps/lib.star

```diff
@@ -775,11 +775,34 @@ def lint_frontend_step():
         "commands": [
             "yarn run prettier:check",
             "yarn run lint",
-            "yarn run i18n:compile",  # TODO: right place for this?
             "yarn run typecheck",
         ],
     }
 
+def verify_i18n_step():
+    extract_error_message = "Extraction failed. Make sure that you have no dynamic translation phrases, such as 't(\\`preferences.theme.\\$${themeID}\\`, themeName)' and that no translation key is used twice."
```

Comment:
What do you think about adding a newline at the beginning of this message (I presume `\n` at the beginning will do the job) to help make the help message stand out a bit more?

Also annoying that the output makes it difficult to see which file has the error - maybe we could add an instruction to the file like `Search the output for `[warning] to find the offending file.`

Reply:
I agree, I do however think that some of this error handling/messages shouldn't be in the CI, but rather directly in the commands, so the help can be displayed as early as possible. But we can go with the fix you mentioned for now.